modules/machineset-modifying.adoc

```diff
@@ -122,14 +201,30 @@ $ oc scale --replicas=2 \// <1>
 +
 [source,terminal]
 ----
-$ oc describe machine <machine_name_updated_1> -n openshift-machine-api
+tag::CAPI[]
+$ oc describe machine.cluster.x-k8s.io <machine_name_updated_1> \
+  -n openshift-cluster-api
+end::CAPI[]
+tag::MAPI[]
+$ oc describe machine.machine.openshift.io <machine_name_updated_1> \
+  -n openshift-machine-api
+end::MAPI[]
 ----
 
 * To verify that the compute machines without the updated configuration are deleted, list the machines that are managed by the updated compute machine set by running the following command:
 +
 [source,terminal]
 ----
-$ oc get -n openshift-machine-api machines -l machine.openshift.io/cluster-api-machineset=<machine_set_name>
+tag::CAPI[]
+$ oc get machines.cluster.x-k8s.io \
+  -n openshift-cluster-api \
+  -l cluster.x-k8s.io/cluster-api-machineset=<machine_set_name>
+end::CAPI[]
+tag::MAPI[]
+$ oc get machines.machine.openshift.io \
+  -n openshift-machine-api \
+  -l machine.openshift.io/cluster-api-machineset=<machine_set_name>
+end::MAPI[]
 ----
 +
 .Example output while deletion is in progress for an AWS cluster
```

Comment:
the result is different with machine api, it looks like 
```
$ oc get machines.cluster.x-k8s.io -n openshift-cluster-api                                                                                                
NAME            CLUSTER             NODENAME                                    PROVIDERID                              PHASE      AGE     VERSION
capi-ms-z4bqv   zhsun-aws29-8ksnk   ip-10-0-31-228.us-east-2.compute.internal   aws:///us-east-2a/i-04e7b2cbd61fd2075   Running    18m
```

Reply:
Thanks! I attempted to expand it to show multiple (will be in an update I push later today)